Tests/ApolloServerIntegrationTests/StarWarsServerCachingRoundtripTests.swift

```diff
@@ -61,9 +62,7 @@ class StarWarsServerCachingRoundtripTests: XCTestCase, CacheDependentTesting {
   }
   
   func testHeroAndFriendsNamesWithIDsQuery() {
-    client.store.cacheKeyForObject = { $0["id"] }
```

Comment:
This doesn't appear to have any affect on the test and the mechanism for cache key resolution has changed to be class-based.